AlfrescoApp/Model/Managers/RealmSyncManager.m

```diff
@@ -1646,7 +1650,60 @@ - (void)startNetworkOperations
         SyncOperationQueue *syncOpQ = [self currentOperationQueue];
         [syncOpQ downloadContentsForNodes:self.nodesToDownload withCompletionBlock:nil];
         [syncOpQ uploadContentsForNodes:self.nodesToUpload withCompletionBlock:nil];
+        
+        [self trackSyncRunWithNodesToDownload:self.nodesToDownload nodesToUpload:self.nodesToUpload];
     });
 }
 
+#pragma mark - Analytics
+
+- (void)trackSyncRunWithNodesToDownload:(NSArray *)nodesToDownload nodesToUpload:(NSArray *)nodesToUpload
+{
+    __block NSUInteger numberOfFiles = 0;
+    __block NSUInteger numberOfFolders = 0;
+    __block unsigned long long totalFileSize = 0;
+    
+    [nodesToDownload enumerateObjectsUsingBlock:^(AlfrescoNode *node, NSUInteger idx, BOOL * stop){
+        if (node.isDocument)
```

Comment:
Since the block is the same in both cases it could be declared only once and used as a parameter.